core/src/main/java/com/microsoft/applicationinsights/internal/schemav2/Envelope.java

```diff
@@ -37,10 +37,11 @@
 public class Envelope
     implements JsonSerializable
 {
+    public static final int DEFAULT_VER = 1;
```

Comment:
Well these all changes will need to go to bond! @littleaj it would be great if you can make a not of what all you modified in bond generated classes and then we can get to address and fix bond scripts so next time a schema rollout happens we do not mess up.

Reply:
Another option is to have a separate class which contains all the same information, but that just creates another file we have to update every time we regenerate the bond files.

Reply:
reverted this change